pkg/runner/local_docker.go

```diff
@@ -136,14 +138,24 @@ func (r *LocalDockerRunner) Healthcheck(ctx context.Context, engine api.Engine,
 		ow.Warnf("guessing docker socket as %s", dockerSock)
 	}
 
+	additionalHosts := "ADDITIONAL_HOSTS="
+	envHosts, hasHosts := engine.EnvConfig().Runners["local:docker"]["additional_hosts"].([]interface{})
+	if hasHosts {
+		for i, host := range envHosts {
+			if i > 0 {
+				additionalHosts += ","
+			}
+			additionalHosts += host.(string)
+		}
+	}
```

Comment:
Could we simplify this piece with the following?
```suggestion
	envHosts, hasHosts := engine.EnvConfig().Runners["local:docker"]["additional_hosts"].([]string)
	if hasHosts {
		additionalHosts += strings.Join(envHosts, ",")
	}
```

Reply:
Good idea! The suggestion has been applied.